pandas/tests/groupby/aggregate/test_cython.py

```diff
@@ -41,7 +41,10 @@
         "max",
     ],
 )
-def test_cythonized_aggers(op_name):
+def test_cythonized_aggers(op_name, using_array_manager):
+    if using_array_manager and op_name in {"count", "sem"}:
+        # TODO(ArrayManager) groupby count/sem
+        pytest.skip("ArrayManager groupby count/sem not yet implemented")
```

Comment:
can the add_marker pattern be used here?

Reply:
We use the `add_marker` pattern for xfail (because just raising pytest.xfail wouldn't result in a strict xfail, unlike the skip here), so for skip there is no advantage using it AFAIK. 

Now, that said, I should maybe actually start using xfail instead of skip for the "skip_array_manager_not_yet_implemented", so it's easier to notice if certain tests can be unskipped when more features get implemented.

So will already start using the xfail as ou suggest here.

Reply:
Actually, in the meantime I fixed `count`, so the skip/xfail could be removed altogether. 

(but a sign that using xfail instead of skip is actually a good idea ;))

Reply:
sounds good